src/test/java/uk/gov/hmcts/reform/sscs/helper/mapping/ServiceHearingValuesMappingTest.java

```diff
@@ -179,7 +178,9 @@ void shouldMapServiceHearingValuesSuccessfully() {
             "hearingLoop",
             "disabledAccess"
         ), serviceHearingValues.getFacilitiesRequired());
-        assertThat(serviceHearingValues.getListingComments()).isEqualToNormalizingNewlines(NOTE_FROM_OTHER_APPELLANT + NOTE_FROM_OTHER_APPELLANT + "\n" + "\n" + NOTE_FROM_OTHER_PARTY + NOTE_FROM_OTHER_PARTY);
+        assertThat(serviceHearingValues.getListingComments())
```

Comment:
Medium: This test has so many asserts that makes it hard to handle failures. It will stop on first failure, so all other tests wont run until first failure dealt with. How can this be broken into several tests so a single run highlights many failures instead of just one?

Reply:
This is already covered by tech debit [SSCS-10565](https://tools.hmcts.net/jira/browse/SSCS-10565)